addons/resource.language.en_gb/resources/strings.po

```diff
@@ -14143,19 +14152,19 @@ msgstr ""
 #. Description of setting "Appearance -> File lists -> Show file extensions" with label #497
 #: system/settings/settings.xml
 msgctxt "#36123"
-msgid "Show file extensions on media files. For example, 'You Enjoy Myself.mp3' would be simply be shown as 'You Enjoy Myself'."
+msgid "Show file extensions on media files. For example, \"You Enjoy Myself.mp3\" would be simply be shown as \"You Enjoy Myself\"."
 msgstr ""
 
 #. Description of setting "Appearance -> File lists -> Ignore articles when sorting with label #13399
 #: system/settings/settings.xml
 msgctxt "#36124"
-msgid "Ignore certain tokens (e.g. \"the\") during sort operations. For example, 'The Simpsons' would be sorted as 'Simpsons'."
+msgid "Ignore certain tokens (for example, \"The\") during sort operations. For example, \"The Simpsons\" would be sorted as \"Simpsons\"."
```

Comment:
No. reason is, because after a full stop it  calls for E.g. and E.g is incorrect, the only format is e.g. or e.g., if US or Canada.

Imo starting a Sentence with with e.g. is just worthy of death. so For example was a good compromise.